projects/Mallard/src/components/article/article-image.tsx

```diff
@@ -1,18 +1,23 @@
 import React from 'react'
 import { StyleSheet, Image } from 'react-native'
-
+import { APIPaths } from 'src/paths'
+import { Image as ImageT } from '../../common'
 const styles = StyleSheet.create({
     image: {
         width: '100%',
         backgroundColor: 'rgba(0,0,0,.25)',
     },
 })
 
-const ArticleImage = ({ image, style }: { image: string; style?: {} }) => (
+const ArticleImage = ({ image, style }: { image: ImageT; style?: {} }) => (
     <Image
         style={[styles.image, style]}
         source={{
-            uri: image,
+            uri: `https://d2mztzjulnpyb8.cloudfront.net/${APIPaths.media(
```

Comment:
can we put this in `src/paths`?

Reply:
yeah that's a much better home for it

Reply:
other than that the app bits looks good!

Reply:
thanks